crates/host/src/wasmbus/mod.rs

```diff
@@ -1933,7 +1933,7 @@ impl Host {
                 let mut queue = Abortable::new(queue, queue_abort_reg);
                 queue
                     .by_ref()
-                    .for_each({
+                    .for_each_concurrent(None, {
```

Comment:
It honestly at this time does the same thing since starting an actor goes into a background task.

To hopefully address your point without nitpicking the example, e.g. the host receives a `stop_actor` message and get inventory at the same time, yes you're right that we now have to consider that we don't have a FIFO queue style of handling CTL requests now

Reply:
It honestly at this time does the same thing since starting an actor goes into a background task.

To hopefully address your point without nitpicking the example, e.g. the host receives a `stop_actor` message and get inventory at the same time, yes you're right that we now have to consider that we don't have a FIFO queue style of handling CTL requests now

Reply:
I really think we have to revert this change in the future, but I'm fine with it for now as a hotfix